bettertest-kafka/src/main/java/org/bitintegro/bettertest/modules/kafka/testclient/annotation/WithKafkaTestClient.java

```diff
@@ -0,0 +1,15 @@
+package org.bitintegro.bettertest.modules.kafka.testclient.annotation;
+
+import org.bitintegro.bettertest.modules.kafka.testclient.springboot.KafkaTestClientConfig;
+import org.springframework.context.annotation.Import;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Import(KafkaTestClientConfig.class)
+public @interface WithKafkaTestClient {
```

Comment:
Please add JavaDoc. 

Please consider to allow to use the annotation with a field from the test class (similar to `@Mock` or `@MockBean` annotations from the Mockito library.)